### ISSUE TITLE
Cannot build rust when nested message contains repeated field

### ISSUE THREAD

**Issue by MyonKeminta** [pingcap/kvproto#623]
In this commit of my PR: https://github.com/pingcap/kvproto/pull/622/commits/e23c6bb183d4af51cceaf020957494d27b614294

And CI failed on building rust:

```
error[E0433]: failed to resolve: `super` in paths can only be used in start position
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:385:65
    |
385 | #[inline] pub fn set_locks(&mut self, v: ::std::vec::Vec<event::super :: super :: kvrpcpb :: LockInfo>) { self.locks = v; }
    |                                                                 ^^^^^ `super` in paths can only be used in start position

error[E0433]: failed to resolve: `super` in paths can only be used in start position
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:386:47
    |
386 | #[inline] pub fn get_locks(&self) -> &[event::super :: super :: kvrpcpb :: LockInfo] { &self.locks }
    |                                               ^^^^^ `super` in paths can only be used in start position

error[E0433]: failed to resolve: `super` in paths can only be used in start position
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:387:70
    |
387 | #[inline] pub fn mut_locks(&mut self) -> &mut ::std::vec::Vec<event::super :: super :: kvrpcpb :: LockInfo> { &mut self.locks }
    |                                                                      ^^^^^ `super` in paths can only be used in start position

error[E0433]: failed to resolve: `super` in paths can only be used in start position
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:388:66
    |
388 | #[inline] pub fn take_locks(&mut self) -> ::std::vec::Vec<event::super :: super :: kvrpcpb :: LockInfo> { ::std::mem::replace(&mut self.locks, ::std::vec::Vec::new()) }
    |                                                                  ^^^^^ `super` in paths can only be used in start position

error[E0412]: cannot find type `u64` in module `change_data_request`
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:595:86
    |
595 | #[inline] pub fn set_txn_start_ts(&mut self, v: ::std::vec::Vec<change_data_request::u64>) { self.txn_start_ts = v; }
    |                                                                                      ^^^ not found in `change_data_request`
    |
help: possible candidates are found in other modules, you can import them into scope
    |
2   | use core::primitive::u64;
    |
2   | use std::primitive::u64;
    |

error[E0412]: cannot find type `u64` in module `change_data_request`
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:596:68
    |
596 | #[inline] pub fn get_txn_start_ts(&self) -> &[change_data_request::u64] { &self.txn_start_ts }
    |                                                                    ^^^ not found in `change_data_request`
    |
help: possible candidates are found in other modules, you can import them into scope
    |
2   | use core::primitive::u64;
    |
2   | use std::primitive::u64;
    |

error[E0412]: cannot find type `u64` in module `change_data_request`
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:597:91
    |
597 | #[inline] pub fn mut_txn_start_ts(&mut self) -> &mut ::std::vec::Vec<change_data_request::u64> { &mut self.txn_start_ts }
    |                                                                                           ^^^ not found in `change_data_request`
    |
help: possible candidates are found in other modules, you can import them into scope
    |
2   | use core::primitive::u64;
    |
2   | use std::primitive::u64;
    |

error[E0412]: cannot find type `u64` in module `change_data_request`
   --> /home/circleci/project/target/debug/build/kvproto-dc3c32c98dc79712/out/protos/wrapper_cdcpb.rs:598:87
    |
598 | #[inline] pub fn take_txn_start_ts(&mut self) -> ::std::vec::Vec<change_data_request::u64> { ::std::mem::replace(&mut self.txn_start_ts, ::std::vec::Vec::new()) }
    |                                                                                       ^^^ not found in `change_data_request`
    |
help: possible candidates are found in other modules, you can import them into scope
    |
2   | use core::primitive::u64;
    |
2   | use std::primitive::u64;
    |

error: aborting due to 8 previous errors
```

**Comment by MyonKeminta** [comment]
@nrc  PTAL thanks!

**Comment by nrc** [comment]
This is fixed by https://github.com/tikv/protobuf-build/pull/41. I'll publish a new version of protobuf-build once it lands

**Comment by nrc** [comment]
I have published protobuf-build 0.11.2

**Comment by MyonKeminta** [comment]
Thank you!
